src/components/side-nav/SideNav.jsx

```diff
@@ -10,6 +11,7 @@ import LikedSongsIcon from "../../assets/icons/LikedSongsIcon";
 import HomeIcon from "../../assets/icons/HomeIcon";
 import SearchIconActive from "../../assets/icons/SearchIconActive";
 import LegalLink from "./components/LegalLink";
+import rootStore from "../../store/root-store";
```

Comment:
Stores that are contained by the root store will also hold a reference to the root store so that they can essentially reference any other store contained by the root store.